### DISCUSSION TITLE
خطا در نصب آخرین نسخه لحظه ای

### DISCUSSION THREAD

**Discussion by mhm322** [kianahmadian/python-bale-bot#116] [Q&A] [answered]
سلام، من وقتی میخوام از آخرین نسخه این کتابخونه استفاده کنم و دستوری که توی سایت گذاشتید رو میزنم، این خطا رو میده

ERROR: Invalid requirement: 'python-bale-bot==any': Expected end or semicolon (after name and no valid version specifier)
    python-bale-bot==any
                   ^

**Comment by kian-ahmadian** [comment]
سلام عزیز وقت بخیر
با نظر به [سورس کد وبسایت](https://github.com/python-bale-bot/python-bale-bot.github.io/blob/master/index.html#L71)، دستوری که قرار داده شده

```
pip install python-bale-bot
```
هست. لطفا مجدد تلاش کنید و در صورتی که کماکان با این مشکل روبه‌رو بودین، بفرمایین

**Reply by mhm322** [reply]
منظورم آخرین نسخه stable نیست، چون میخواستم از handler ها استفاده کنم
![image](https://github.com/user-attachments/assets/22785b27-d6a1-4d2e-9315-48cf33dcad63)

**Reply by kian-ahmadian** [reply, marked as answer]
متوجه منظورتون شدم. در حال حاضر امکان نصب نسخه لحظه ای به این دلیل که توسعه دهندگان کتابخانه، نتایج نهایی بخش های مختلف را ارائه نکرده اند وجود ندارد. انشالله در روز های آینده با میسر شدن این امکان در این قسمت اطلاع رسانی خواهیم کرد
